ex_34.py

amymhaddad opened the pull request:
Create a small program that contains a list of employee names. Print out the list of names when the program runs the first time. Prompt for an employee name and remove that specific name from the list of names. Display the remaining employees, each on its own line.

amymhaddad commented:
@paulghaddad I'm ignoring hound's comment on line length.

paulghaddad commented on ex_34.py:
```diff
@@ -6,24 +6,27 @@
-    print(name.capitalize())
+# Step 2: print the total number of employees and print their names
+print(f"There are {total_employees} employees: ")
+for name in employee_names:
```
I'd add a blank line to separate the variable from the loop.

paulghaddad commented on ex_34.py:
```diff
@@ -6,24 +6,27 @@
+# Step 2: print the total number of employees and print their names
+print(f"There are {total_employees} employees: ")
+for name in employee_names:
+    print(name.title())
```
👏

paulghaddad commented on ex_34.py:
```diff
@@ -6,24 +6,27 @@
-names_lowercase.remove(employee_removal)
+# Step 3: get user input on employee removal and remove employee
+employee_removal = input("Which employee should I remove? ")
+normalized_employee_removal = employee_removal.lower()
```
Suggest: `employee_to_remove` and `normalized_employee_to_remove`.

amymhaddad commented:
@paulghaddad Added the space and changed the variable names!

amymhaddad commented:
@paulghaddad I caught an error when I was re-doing this exercise: I needed to use "count" for a second time at the end of the program to calculate the remaining employees. I added that in and added a constraint: a conditional if user enters invalid input.
